### ISSUE TITLE
Request: move CpoyButton lower

### ISSUE THREAD

**Issue by Jay-o-Way** [microsoft/WinUI-Gallery#1533]
### Issue type

enhancement

### Which version of the app?

WinUI 3 Gallery

### Description

Currently, the CopyButton is usually displayed on one line, next to the related value. However, for the colors, it's displayed in the top-right corner, while the value is on the bottom. It would be nice if the CopyButton sticks immediately to the right of the value everywhere.

### Screenshots

![image](https://github.com/microsoft/WinUI-Gallery/assets/65828559/1fbffb46-a228-4728-a53e-6b9406601901)

### Windows version

Windows 11 23H2 (22631)

### Additional context

Check validity / status for this message.
![image](https://github.com/microsoft/WinUI-Gallery/assets/65828559/104e94ac-cd07-4892-8871-1692a3ef3081)

**Comment by Jay-o-Way** [comment]
I can create a PR for this 💪

**Comment by niels9001** [comment]
Understand the reasoning! But we moved it to the top because the brush names tend to be much longer than the titles. As a result, brush names are either clipped or wrapped, both not great.

**Comment by karkarl** [comment]
Is there a reason why the copy buttons don't only show up on hover? That might solve the issue with longer brush names. I remember reading a Microsoft design guideline that discourages it.

**Comment by niels9001** [comment]
> Is there a reason why the copy buttons don't only show up on hover? That might solve the issue with longer brush names. I remember reading a Microsoft design guideline that discourages it.

That would require adding keyboarding support as well :(

**Comment by Jay-o-Way** [comment]
> we moved it to the top because the brush names tend to be much longer than the titles. As a result, brush names are either clipped or wrapped, both not great.

That argument is not valid in every case:
![image](https://github.com/microsoft/WinUI-Gallery/assets/65828559/7f2e5c46-2c6d-489a-b9e9-139addb7db4f)

**Comment by Jay-o-Way** [comment]
@karkarl can you confirm if the message is outdated and can be removed?

>This brush might not (yet) be available in WinUI

**Comment by niels9001** [comment]
I don't think we cam up with a better solution, so closing this issue for now
